scripts/solve_field.sh

```diff
@@ -6,7 +6,7 @@ then
   echo 2>&1 ""
   echo 2>&1 "Please install astrometry into ${PANDIR}/astrometry to enable $0"
   echo 2>&1 ""
-  exit 1
+  exit 127 # Command not found
```

Comment:
I don't actually know if this is the appropriate use of the error code 127 so don't mind switching back to `1` as long as we still catch above.

Reply:
You might be better to go with 3. Codes 1, 2, 126 and 127 have meanings assigned by bash, so avoiding those codes may avoid confusion down the road. See: http://tldp.org/LDP/abs/html/exitcodes.html